recipe/yum_requirements.txt

```diff
@@ -1,4 +1,5 @@
 libXt-devel
 libXmu-devel
 libXi-devel
-mesa-libGLU-devel
\ No newline at end of file
+mesa-libGLU-devel
+mesa-libGL
```

Comment:
It would be good to get rid of (all) this in favour of `libglvnd` resp the various xorg packages.

Reply:
I think we should start with this in the coin3d-feedstock:
https://github.com/conda-forge/coin3d-feedstock